sdk/ml/azure-ai-ml/azure/ai/ml/_artifacts/_blob_storage_helper.py

```diff
@@ -90,8 +91,9 @@ def upload(
             # warn if large file (> 100 MB)
             file_size, _ = get_directory_size(source)
             file_size_in_mb = file_size / 10**6
+            full_storage_url = f"https://{self.account_name}.blob.core.windows.net/{self.container}/{dest}"
```

Comment:
.blob.core.windows.net won't work in non public cloud.

Reply:
What would the change need to be for private clouds? If you have a short explanation, I can add a note to the message.

Reply:
Addressed offline and fixed in code.